WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -106,18 +106,32 @@ class StoreOnboardingViewModel: ObservableObject {
 
 private extension StoreOnboardingViewModel {
     @MainActor
-    private func loadTasks() async throws -> [StoreOnboardingTaskViewModel] {
-        try await withCheckedThrowingContinuation { continuation in
+    func loadTasks() async throws -> [StoreOnboardingTaskViewModel] {
+        let shouldManuallyAppendLaunchStoreTask = await isFreeTrialPlan()
+        let tasksFromServer: [StoreOnboardingTask] = try await withCheckedThrowingContinuation({ continuation in
```

Comment:
nit: can we use `async let` or a task group to perform these two async tasks in parallel?